lib/chorde/clients/memcached.py

```diff
@@ -1478,7 +1478,7 @@ def method(k):
                 page_keys = [ page_prefix + ascii(i) for i in range(1,npages) ]
```

Comment:
This is where the encode needs to happen, `page_prefix` is bytes (if you check):
```suggestion
                page_keys = [ page_prefix + ascii(i).encode("ascii") for i in range(1,npages) ]
```